tests/test_data/sii-cte/f29/cte--61002000-3--f29-6600000016-datos-obj-fake-missing-code-49.json

```
@@ -0,0 +1,220 @@
+      "FECHA_INGRESO": "28082017",
+      "FOLIO": "6600000016",
+      "MEDIO_INGRESO": "INTERNET",
+      "MEDIO_PAGO": "PEL",
+      "MOSTRAR_FOLIO": "1",
+      "NOMBRES": " SERVICIO DE REGISTRO CIVIL E IDENTIFICACION ",
+      "NOMBRE_FUN": "  ",
+      "OPCION": "VC",
+      "PERIODO": "201707",
+      "RUT_ORIGEN": "61002000",
+      "TIPO_MOVIMIENTO": "1",
+      "TOKEN": "VFC",
+      "USUARIO": "0",
+      "cant_anuladas": 0
+    },
+    "folioF": {},
+    "glosa": {
+      "001": "APELLIDO PATERNO O RAZ\u00d3N SOCIAL",
+      "003": "N DE RUT",
+      "006": "DIRECCION",
+      "007": "FOLIO",
+      "008": "COMUNA",
+      "015": "PERIODO TRIBUTARIO",
+      "030": "PPM ART. 84, A) PERD. ART. 90",
+      "048": "RET. IMP. \u00daNICO TRAB. ART. 74 N 1 LIR",
+      "060": "PORCENTAJE CONDONACION TGR",
+      "077": "REMANENTE DE CR\u00c9DITO FISC.",
+      "089": "IMP. DETERM. IVA DETERM.",
+      "091": "TOTAL A PAGAR DENTRO DEL PLAZO",
+      "092": "REAJUSTES",
+      "093": "Intereses y multas",
+      "094": "Total a pagar con recargo",
+      "151": "RET, TASAS DE 10 % SOBRE LAS RENT.  ",
+      "315": "FECHA TIMBRE CAJA",
+      "502": "D\u00c9BITOS FACTURAS EMITIDAS",
+      "503": "CANTIDAD FACTURAS EMITIDAS",
+      "504": "REMANENTE CREDITO MES ANTERIOR",
+      "511": "CR\u00c9D. IVA POR DCTOS. ELECTRONICOS",
+      "519": "CANT. DE DCTOS. FACT. RECIB. DEL GIRO",
+      "520": "CR\u00c9DITO REC. Y REINT./FACT. DEL GIRO",
+      "527": "CANT. NOTAS DE CR\u00c9DITO RECIBIDAS",
+      "528": "CR\u00c9DITO RECUP. Y REINT NOTAS DE CR\u00c9D",
+      "537": "TOTAL CR\u00c9DITOS",
+      "538": "TOTAL D\u00c9BITOS",
+      "547": "TOTAL DETERMINADO",
+      "562": "MONTO SIN DER. A CRED. FISCAL",
+      "584": "CANT.INT.EX.NO GRAV.SIN DER. CRED.FISCAL",
+      "595": "SUB TOTAL IMP. DETERMINADO ANVERSO",
+      "761": "CANT. FACT. SUPERMERCADOS Y SIMILARES",
+      "762": "CR\u00c9D. FACT. SUPERMERCADOS Y SIMILARES",
+      "795": "MONTO DE CONDONACION SII",
+      "915": "Fecha de Vigencia de Condonacion",
+      "922": "NUMERO RESOLUCION SII AUTO. CONDONACION"
+    },
+    "justif": {
+      "001": "I",
+      "003": "I",
+      "006": "I",
+      "007": "I",
+      "008": "I",
+      "015": "I",
+      "030": "I",
+      "048": "I",
+      "060": "I",
+      "077": "I",
+      "089": "D",
+      "091": "-",
+      "092": "-",
+      "093": "-",
+      "094": "-",
+      "151": "I",
+      "315": "I",
+      "502": "D",
+      "503": "I",
+      "504": "D",
+      "511": "I",
+      "519": "I",
+      "520": "D",
+      "527": "I",
+      "528": "D",
+      "537": "D",
+      "538": "D",
+      "547": "D",
+      "562": "D",
+      "584": "I",
+      "595": "D",
+      "761": "I",
+      "762": "D",
+      "795": "I",
+      "915": "-",
+      "922": "-"
+    },
+    "linea": {
+      "001": "0",
+      "003": "0",
+      "006": "0",
+      "007": "0",
+      "008": "0",
+      "015": "0",
+      "030": "56",
+      "048": "49",
```

The real data that we have has this.

```suggestion
      "048": "49",
      "049": "0",
```